conf/routes

```diff
@@ -79,7 +79,7 @@ POST           /api/authenticate/:provider          @com.yetu.oauth2provider.con
 # New registration page
 GET            /setup/registration                  @com.yetu.oauth2provider.controllers.setup.SetupController.startSignUp
 POST           /setup/registration                  @com.yetu.oauth2provider.controllers.setup.SetupController.handleStartSignUp
-
+POST           /setup/checkPassword                 @com.yetu.oauth2provider.controllers.setup.SetupController.checkPassword
```

Comment:
I´m very happy, that is is done now. Big thanks to @larsblumberg and @ghophp. It looks fine for me. I have not tested it yet, but I will do it on the weekend, but I think:
LGTM!